ui/app/components/mount-accessor-select.js

```diff
@@ -1,34 +1,49 @@
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import Component from '@ember/component';
-import { task } from 'ember-concurrency';
+import { dropTask } from 'ember-concurrency';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  store: service(),
+/**
+ * @module MountAccessorSelect
+ * The MountAccessorSelect component is used to selectDrop down mount options.
+ *
+ * @example
+ * ```js
+ * <MountAccessorSelect @value={this.aliasMountAccessor} @onChange={this.onChange} />
+ * ```
+ * @param {string} value - the selected value.
+ * @param {function} onChange - the parent function that handles when a new value is selected.
+ * @param {boolean} [showAccessor] - whether or not you should show the value or the more detailed accessor off the class.
+ * @param {boolean} [noDefault] - whether or not there is a default value.
+ * @param {boolean} [filterToken] - whether or not you should filter out type "token".
+ * @param {string} [name] - name on the label.
+ * @param {string} [label] - label above the select input.
+ * @param {string} [helpText] - text shown in tooltip.
+ */
 
-  // Public API
-  //value for the external mount selector
-  value: null,
-  filterToken: false,
-  noDefault: false,
-  onChange: () => {},
+export default class MountAccessorSelect extends Component {
+  @service store;
 
-  init() {
-    this._super(...arguments);
+  filterToken = false;
```

Comment:
Since these attrs have default values, would it be better to use getters here? ([docs](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_arguments))